packages/@aws-cdk/aws-glue-alpha/lib/job.ts

```diff
@@ -719,6 +719,16 @@ export class Job extends JobBase {
       }
     }
 
+    if (props.maxCapacity !== undefined && (props.workerType || props.workerCount !== undefined)) {
```

Comment:
Looking for some clarification on this - the documentation says that `maxCapacity` cannot be used when setting `workerType` and `workerCount`. Why are we only checking for `workerType` or `workerCount` instead of `workerType` and `workerCount`?

Reply:
@colifran 

Thanks for your review!

I was really worried.

If "and" is used, in the case where `maxCapacity` is specified but only one of `workerType` or `workerCount` is specified, it will break through this branch and throw the error `Both workerType and workerCount must be set`.
I did this because I thought that the essential error, the collision between `maxCapacity` and `workerType `, should be fixed first.

But it's certainly possible that there could be some confusion due to the deviation from the document, so I'll fix it!